clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/StateTransferBuilder.java

```diff
@@ -22,8 +22,7 @@
 
 package org.jboss.as.clustering.infinispan.subsystem;
 
-import static org.jboss.as.clustering.infinispan.subsystem.StateTransferResourceDefinition.Attribute.CHUNK_SIZE;
-import static org.jboss.as.clustering.infinispan.subsystem.StateTransferResourceDefinition.Attribute.TIMEOUT;
+import static org.jboss.as.clustering.infinispan.subsystem.StateTransferResourceDefinition.Attribute.*;
```

Comment:
could you use explicit import instead of `*`?

Reply:
In general, a given ResourceServiceBuilder always references all values of the associated Attribute enumeration, so the explicit import never excludes anything.  But, I will fix.